README.md

```diff
@@ -16,20 +16,30 @@ following new functionality:
 
 ## Getting Started and Documentation
 
-Follow these steps to get started with `provider-argocd`.
+Follow the [official docs](https://crossplane.io/docs/master/getting-started/install-configure.html#install-crossplane) to install crossplane, then these steps to get started with `provider-argocd`.
 
 ### Optional: Start a Local ArgoCD Server
+```bash
+kind create cluster
 
-    kind create cluster
-
-    kubectl create ns argocd
-
-    kubectl apply -n argocd --force -f https://raw.githubusercontent.com/argoproj/argo-cd/release-2.0/manifests/install.yaml
-
+kubectl create ns argocd
+kubectl apply -n argocd --force -f https://raw.githubusercontent.com/argoproj/argo-cd/stable/manifests/install.yaml
```

Comment:
Is `--force` necessary?

Reply:
I checked the argocd docs and there is no `--force` anymore so this must be a left-over